package.json

```diff
@@ -30,7 +30,5 @@
   },
   "license": "Apache-2.0",
   "typings": "./js/index.d.ts",
-  "dependencies": {
-    "npm": "^5.8.0"
```

Comment:
do we still want the npm dependency?

Reply:
It isn't necessary, and was exposing us to some security vulnerabilities of its own.

Reply:
No idea why it was in there.